src/uct/ib/rdmacm/rdmacm_cm.h

```diff
@@ -8,6 +8,14 @@
 #include "rdmacm_def.h"
 
 
+/**
+ * RDMACM CM configuration.
+ */
+typedef struct uct_rdmacm_cm_config {
+    uct_cm_config_t  super;
+} uct_rdmacm_cm_config_t;
```

Comment:
why need to define config struct for rdmacm_cm in this PR?
there are no fields here
just use base config at this point